internal/api/commitment_test.go

```diff
@@ -542,3 +560,211 @@ func TestDeleteCommitmentErrorCases(t *testing.T) {
 		ExpectBody:   assert.StringData("no such commitment\n"),
 	}.Check(t, s.Handler)
 }
+
+func Test_StartCommitmentTransfer(t *testing.T) {
+	s := test.NewSetup(t,
+		test.WithDBFixtureFile("fixtures/start-data-commitments.sql"),
+		test.WithConfig(testCommitmentsYAMLWithoutMinConfirmDate),
+		test.WithAPIHandler(NewV1API),
+	)
+
+	var transferToken = test.GenerateDummyToken()
+
+	// Test on confirmed commitment should succeed.
+	// TransferAmount >= CommitmentAmount
+	req1 := func(transfer_status string) assert.JSONObject {
```

Comment:
Go does not usually use snake_case:
```suggestion
	req1 := func(transferStatus string) assert.JSONObject {
```